### PR TITLE
Add something to test on testwiki

### PR DESCRIPTION
**Pull request by Reception123**
I would like to test something on the testwiki.

### CODE REVIEW

File: roles/mediawiki/files/localsettings/LocalSettings.php.j2 [hunk 1]
```diff
@@ -1978,6 +1983,7 @@ $wgConf->settings = array(
 	'wgAllowUserCss' => array(
 		'default' => true,
 	),
+	'wgRestrictionLevels' => array( 'testwiki' => 'bureaucratprotect' ),
```
Comment (addshore) [inline roles/mediawiki/files/localsettings/LocalSettings.php.j2]: this should really be split onto multiple lines

Comment (NDKilla) [inline roles/mediawiki/files/localsettings/LocalSettings.php.j2]: This actually needs to be removed. It is replacing the setting already at line 1766
I don't know why it was added twice

File: roles/mediawiki/files/localsettings/LocalSettings.php.j2 [hunk 2]
```diff
@@ -1977,7 +1982,6 @@ $wgConf->settings = array(
 	),
 	'wgAllowUserCss' => array(
 		'default' => true,
-	),
```
Comment (addshore) [inline roles/mediawiki/files/localsettings/LocalSettings.php.j2]: This line should now be removed..

**Comment by addshore** [comment]
Also we are now at 3 commits for a 5 line change....
Please SQUASH THE COMMITS and FORM PUSH TO YOUR BRANCH
or I will not merge this.
PRs like this are just getting silly.....

**Comment by NDKilla** [comment]
Allow user css actually needs to be closed.. Tbh the first commit was build passing and did everything right. Could you merge only the first commit?

**Comment by addshore** [comment]
on Github, no, we cant not only merge the first commit.
You need to fix the branch in git and repush please!

**Comment by addshore** [comment]
alternatively fix in a new PR >.>

**Comment by Reception123** [comment]
I will take care of this later.

**Pull request closed without merging**